hopsworks-common/src/main/java/io/hops/hopsworks/common/jobs/spark/SparkYarnRunnerBuilder.java

```diff
@@ -133,7 +133,7 @@ public SparkYarnRunnerBuilder(Jobs job) {
    * @throws IOException If creation failed.
    */
   public YarnRunner getYarnRunner(String project,
-      String jobUser, AsynchronousJobExecutor services,
+      String jobUser, String userName, AsynchronousJobExecutor services,
```

Comment:
From what I get, `userName` is the user's full name? Let's name it `usersFullName` cause we have username in hopsworks, username in hdfs and it can be confusing

Reply:
Yes the full name, changing it.